test/screenshot/.stylelintrc.yaml

```diff
@@ -3,6 +3,7 @@ rules:
   # CSS
   custom-media-pattern: "^(?:test|custom|report)-.+"
   custom-property-pattern: "^(?:test|custom|report)-.+"
+  max-line-length: 120
```

Comment:
I think you probably meant to add this to the root-level `.stylelintrc.yaml`, not just the one under `test/screenshot`?

Reply:
whoops, yes